app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -803,6 +826,39 @@ Vue.mixin({
           vm.fulfillment_context.active_step = 'payout_amount';
           break;
       }
+    },
+    showWorkSubmitted: function(handle) {
+      let vm = this;
+
+      if (
+        vm.contxt.is_staff ||
+        vm.isOwner ||
+        (handle && handle == vm.contxt.github_handle)
+      ) {
+        return true;
+      }
+      return false;
+    },
+    validateNervos: function() {
+      let vm = this;
+
+      const ADDRESS_REGEX = new RegExp('^(ckb){1}[0-9a-zA-Z]{43,92}$');
+      const isValid = ADDRESS_REGEX.test(vm.bounty.bounty_owner_address);
+
+      if (isValid) {
+        return true;
+      }
+
+      return false;
+    },
+    validateFunderAddress: function() {
+      let vm = this;
+
+      vm.errors = {};
+
+      if (!vm.validateNervos()) {
+        vm.$set(vm.errors, 'funderAddress', 'Please enter a valid Nervos address');
+      }
```

Comment:
^ this is related to ckb -> so let's move it to nervos_extension.js

Reply:
i can move `validateNervos` but `validateFunderAddress` is intended to be generic

Reply:
update: `nervos_extension.js` no longer exists (https://github.com/gitcoinco/web/pull/8550/commits/8174c4d64a751ddce132d893375a1659933418a5)... realised it wasn't used anywhere... i retested

Reply:
wait how do you fire a backend call without `nervos_extension.js`  ? :P I can't seem to find that trigger at my end

Reply:
i used the crosschain flow in `details2.html` from L630

it fires backend call from `fulfillmentComplete()` in `bounty_details2.js` from L476